### DISCUSSION TITLE
how to deploy on free platforms

### DISCUSSION THREAD

**Discussion by Rigolino1** [AlistGo/alist#3344] [General]
how to deploy alist on cloudflare workers or VERCEL, or heroku, etc.

**Comment by anwen-anyi** [comment]
VERCEL doesn't know
heroku has been banned from building, and the account will be banned

**Reply by woshichenghaibo** [reply]
实际测试过VERCEL失败了

**Comment by nquyenancu** [comment]
I found it
https://www.youtube.com/watch?v=MoO0fStWoHM
